oauth2client/service_account.py

```diff
@@ -461,3 +465,94 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
+   epoch = datetime.datetime(1970, 1, 1)
+   time_delta = utc_time - epoch
+   return time_delta.days * 86400 + time_delta.seconds
+
+
+class JWTAccessCredentials(GoogleCredentials):
```

Comment:
- These credentials need to be a GoogleCredentials because one of the "Usable Auth" requirements is that ```GoogleCredentials.get_application_default()```  will return a GoogleCredentials object that uses JWT authentication by default if service account credentials are found on the host.
- Not really.  In order to get GoogleCredentials that actually work, we need to pass in an 'aud' claim.  This will mean having an extra optional parameter ```additional_claims``` in the get_access_token (and possibly authorize method).  One of the suggestions was to add that parameter to all GoogleCredentials, but I'm hesitant to do that.
-Concrete: (get_access_token, create_scoped, create_scoped_required, revoke, to_json, _refresh)
-Abstract: None (All abstract methods are made concrete higher up)
-All of them.